sdk/core/azure-core-tracing-opentelemetry/README.md

```diff
@@ -4,46 +4,80 @@
 
 ## Getting started
 
-### Install the package
+You can enable distributes tracing in Azure client libraries by installing configuring the OpenTelemetry SDK.
+OpenTelemetry is a popular open-source observability framework for generating, capturing, and collecting telemetry data for cloud-native software.
 
-Install the Azure Core OpenTelemetry Tracing plugin for Python with [pip](https://pypi.org/project/pip/):
+There are two key concepts related to tracing: span and trace. A span represents a single operation in a trace. A span can represent an HTTP request,
+a remote procedure call (RPC), a database query, or even the path that your code takes. A trace is a tree of spans showing the path of work through
+a system. You can distinguish a trace on its own by a unique 16-byte sequence called a TraceID. For more information on these concepts and how they
+relate to OpenTelemetry, see the [OpenTelemetry documentation](https://opentelemetry.io/docs/).
 
-```bash
-pip install azure-core-tracing-opentelemetry
+## Tracing with Azure Monitor OpenTelemetry Distro
+
+[Azure Monitor OpenTelemetry Distro](https://learn.microsoft.com/azure/azure-monitor/app/opentelemetry-enable?tabs=python) supports tracing for Azure
+SDKs by default. Just install and configure the distro and use Azure clients as usual.
+
+```python
+
+# Enable Azure Monitor OpenTelemetry Distro
+# It confiures Azure SDKs to use OpenTelemetry as well
+from azure.monitor.opentelemetry import configure_azure_monitor
+from opentelemetry import trace
+
+configure_azure_monitor(
+   connection_string="<your-connection-string>"
+)
+
+# Use Azure SDKs as usual, here as an example with Storage SDKs
+# you may also report your own spans for it.
+from azure.storage.blob import BlobServiceClient
+
+tracer = trace.get_tracer(__name__)
+with tracer.start_as_current_span(name="MyApplication"):
+    client = BlobServiceClient.from_connection_string('connectionstring')
+    client.create_container('my_container')  # Call will be traced
 ```
 
-Now you can use OpenTelemetry for Python as usual with any SDKs that are compatible
-with azure-core tracing. This includes (not exhaustive list), azure-storage-blob, azure-keyvault-secrets, azure-eventhub, etc.
+The Azure Monitor OpenTelemetry Distro can be found in the [`azure-monitor-opentelemetry`](https://pypi.org/project/azure-monitor-opentelemetry) package.
 
-## Key concepts
+### Tracing with generic OpenTelemetry
```

Comment:
Keep this section the same level as "Tracing with Azure Monitor OpenTelemetry Distro"?
```suggestion
## Tracing with generic OpenTelemetry
```